util.go

```diff
@@ -9,3 +16,170 @@ func StringPtr(s string) *string {
 func BoolPtr(b bool) *bool {
 	return &b
 }
+
+// containsString returns true if the given list contains the given element.
+func containsString(s []string, e string) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Using `a` and `s` as the variable names is quite confusing for the future reader, can you rename them?

Reply:
Resolved